examples/microwave-oven-app/microwave-oven-common/microwave-oven-app.matter

```diff
@@ -1731,6 +1731,29 @@ endpoint 1 {
     callback attribute clusterRevision;
   }
 
+  server cluster BasicInformation {
```

Comment:
Is basic information required here? 

I would expect this to be a utility cluster on endpoint 0 only. Can we remove and still pass conformance?